pkgs/tools/backup/s3ql/default.nix

```diff
@@ -1,31 +1,48 @@
-{ stdenv, fetchFromGitHub, python3Packages, sqlite, which }:
+{ stdenv, fetchFromGitHub, fetchpatch, python3Packages, sqlite, which, psmisc }:
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, fetchpatch, python3Packages, sqlite, which, psmisc, makeWrapper }:
```

Reply:
Sorry, the remaining makeWrapper was a mistake